### DISCUSSION TITLE
I have no issues; it works perfectly. There’s just one minor thing that’s not a big problem.

### DISCUSSION THREAD

**Discussion by gamer45hg57** [aonez/Keka#1504] [General]
Every time I need to use the Keka app to compress a file into 7z format, I have to open the app because it defaults to ZIP.

**Comment by gingerbeardman** [comment]
Open Keka > Settings and change the default to 7z!

<img width="736" alt="Screen shot 2024-09-06 at 07 42 12" src="https://github.com/user-attachments/assets/0aaf596a-ef96-4d49-80c6-4782d9caf5ed">

You can have a different default when the app is open:

<img width="448" alt="Screen shot 2024-09-06 at 07 43 38" src="https://github.com/user-attachments/assets/f6b95eb4-af0d-480d-9178-9b677b21a57f">
